sync_staging.yml

```diff
@@ -0,0 +1,54 @@
+---
+resources:
+  - name: every-3-hours
+    type: time
+    source:
+      interval: 3h
+
+  - name: osbook
+    type: git
+    source:
+      uri: git@github.com:openstax/osbooks-college-algebra-bundle.git
+      branch: staging
+      private_key: ((github-private-key))
+
+jobs:
+  - name: sync-staging-branch
+    public: true
+    plan:
+    - get: every-3-hours
+      trigger: true
+    - get: osbook
+    - task: sync content
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            username: ((docker-username))
+            password: ((docker-password))
+            repository: bn16/osbook-synchronizer
+        inputs:
+          - name: osbook
+        outputs:
+          - name: updated-osbook
+        run:
+          path: /bin/bash
+          args:
+            - '-cxe'
+            - |-
+
+              git clone osbook updated-osbook
```

Comment:
🤔  - There's probably a reason that's just not occurring to me, but why clone again if the resource will have pulled things down already in `osbook` directory? Basically, do we need the `updated-osbook` output at all versus just working directly in `osbook`?

Reply:
From my knowledge, `get` on a git resource doesn't initialize git - `git init` and I tried this different ways and found the `git clone` [example](https://concoursetutorial.com/basics/publishing-outputs/) to be the cleanest way to do this. 

The error that I was running into was [this one](https://github.com/openstax/cnx/issues/1394#issuecomment-783732132). When I already had the repo in the pipeline, but then trying to push to it, it would get rejected. 

Please let me know if you are able to find an example that may be one the lines of what you're thinking, I believe I was thinking the same thing as well.

(Will do another round of checks for this)

Reply:
Update: I removed clone, by using the `osbook` resource as input and output